changelog/issue-6945.md

```diff
@@ -0,0 +1,6 @@
+audience: users
+level: minor
```

Comment:
Wdyt?

```suggestion
level: patch
```

Reply:
lol, it was a long weekend, i forgot what levels we have 😆